### ISSUE TITLE
Have a problem about the  "Test pretrained model on mp4 video file".

### ISSUE THREAD

**Issue by WillDamon** [metalbubble/TRN-pytorch#2]
@metalbubble 
Firstly, thank you so much for your excellent code and paper.
When I try to repeat your experiment in the step of "Test pretrained model on mp4 video file", the showed result is good, but I can not find the predicted_video.mp4 in the sample_data folder, could you tell me the reason?   
And whether this step support the input video of .h264/.avi?  I try them both, but I can not get the result.
Thank you so much again.(The result is showed as below)

Another question, where can I find the pretrained model "TRN_something_RGB_BNInception_TRN_segment3_best.pth.tar "?   

....../something_something/TRN-pytorch$ python test_video.py --arch InceptionV3 --dataset moments --weight pretrain/TRN_moments_RGB_InceptionV3_TRNmultiscale_segment8_best.pth.tar --video_file sample_data/juggling.mp4 --rendered_output sample_data/predicted_video.mp4
('Multi-Scale Temporal Relation Network Module in use', ['8-frame relation', '7-frame relation', '6-frame relation', '5-frame relation', '4-frame relation', '3-frame relation', '2-frame relation'])
Freezing BatchNorm2D except the first one.
Extracting frames using ffmpeg...
RESULT ON sample_data/juggling.mp4
1.000 -> juggling
0.000 -> catching
0.000 -> balancing
0.000 -> spinning
0.000 -> throwing
[MoviePy] >>>> Building video sample_data/predicted_video.mp4
[MoviePy] Writing video sample_data/predicted_video.mp4
 89%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▉                          | 8/9 [00:00<00:00, 60.99it/s]
[MoviePy] Done.
[MoviePy] >>>> Video ready: sample_data/predicted_video.mp4

**Comment by alexandonian** [comment]
Hi Will,

The output you supplied would suggest that everything worked correctly! In fact, I copy and pasted the command you used, observed exactly the same output and was able to find predicted_video.mp4 inside the sample_data directory. Since you are able to obtain model predictions but not find the predicted_video.mp4, I would suggest verifying that your installation of `MoviePy` and `OpenCV` are working properly and trying again, as these packages are used to render and save the output video. If not, you could always use some other method of rendering the video that has worked for you in the past. Hope this helps!

Alex

**Comment by WillDamon** [comment]
Thank you so much, Alex.
I will verify the installation of MoviePy and OpenCV , then try it again.

But there is one more question, Do you know where can I find the pretrained model "TRN_something_RGB_BNInception_TRN_segment3_best.pth.tar "?

**Comment by alexandonian** [comment]
Hi Will,

We haven't released that pretrained model yet, but we can just as we have done with the others. I can make it available for download shortly, so just keep an eye out for updates to the repo.

Alex

**Comment by WillDamon** [comment]
Hello, Alex
I got that, thank you so much.

Will

**Comment by alexandonian** [comment]
Hi Will,

You should be able to download `TRN_something_RGB_BNInception_TRN_segment3_best.pth.tar` using the `download_models.sh` script.

Best,
Alex

**Comment by WillDamon** [comment]
Hello, Alex
I have downloaded this pretrained model, and it worked, thank you so much for your help and excellent work.

Will

**Comment by 595448755** [comment]
> python test_video.py --arch InceptionV3 --dataset moments --weight pretrain/TRN_moments_RGB_InceptionV3_TRNmultiscale_segment8_best.pth.tar --video_file sample_data/juggling.mp4 --rendered_output sample_data/predicted_video.mp4

Can you tell me how long does the "--rendered_output sample_data/predicted_video.mp4" last?
